src/content/en/2020/fonts.md

```diff
@@ -22,3 +22,266 @@ queries: 04_Fonts
+  sheets_gid="655527383",
+  sql_file="04_10.font_formats.sql"
+  )
+}}
+
+WOFF is an older, less efficient compression mechanism, but almost universally supported, accounting for an additional 11.6% of fonts served. In almost all cases (Internet Explorer 9-11 being the main exception), serving a font as WOFF is leaving performance on the table, and shows a risk of self-hosting. Even if the format choices were optimal at the time of integration, it requires extra effort to update them as browsers improve. Using a hosted service guarantees that the best format is chosen, along with all relevant optimizations.
+
+Ancient versions of Internet Explorer (6-8), which still make about 1.5% of global browser share, require EOT. These don't show up in the top 5 MIME formats, but are necessary for maximum compatibility.
+
+Uncompressed fonts are 2-3x larger than compressed, but still make up almost 5% of all fonts served, disproportionally on mobile. If you're serving these, it should be a red flag that optimization is possible.
+
+## Popular fonts
+
+Icon fonts are fully half of the top 10 most popular web fonts, the rest being clean, robust sans-serif typeface designs (Roboto Slab is at #19 and Playfair Display at #28 in this ranking, for debuts of other styles, though serif designs are well represented in the tail of the distribution).
+
+{{ figure_markup(
+  image="fonts-popular-typefaces.png",
+  caption="Popular typefaces.",
+  description="A bar chart showing the 10 most popular web fonts.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=551344676&format=interactive",
+  sheets_gid="655527383",
+  sql_file="04_11a.popular_typeface.sql"
+  )
+}}
+
+A note of caution, in determining the most popular fonts you can get different results depending on measurement methodology. The chart above is based on counting the number of pages that include an `@font-face` rule referencing the named font. That counts multiple styles only once, which arguably weights in favor of single-style fonts.
+
+## Color fonts
+
+Color fonts, in one form or other, are supported by most modern browsers, but usage is still close to nonexistent (a total of 755 pages total, the majority of which are in a deprecated format). No doubt part of the problem is the diversity of formats, in fact 4 in widespread use. These come in bitmap and vector flavors. The two bitmap formats are technologically very similar, but SBIX (originally a proprietary Apple format) is not supported in Firefox, while CBDT/CBLC is not supported in Safari.
+
+The COLR vector format is supported on all major modern browsers, but only fairly recently. The fourth format is embedding SVG in OpenType (not to be confused with SVG fonts), but not supported in Chrome or Firefox, largely because SVG is a heavyweight format and difficult to render efficiently.
```

Comment:
The comment about SVG being "difficult to render efficiently" is contentious and unsubstantiated, particularly for the SVG Native subset actually used for SVG in OpenType fonts. Suggest changing 

" but not supported in Chrome or Firefox, largely because SVG is a heavyweight format and difficult to render efficiently."

to

" but not supported in Chrome. One drawback of SVG in OpenType is lack of support for font variations, an increasingly important aspect of modern Web design. For this reason, the COLOR format is likely to prevail, particularly as the lack of gradient and clipping support in COLR is now being addressed."